server/images.go

```diff
@@ -119,10 +119,6 @@ func Prompt(promptTemplate string, p PromptVars) (string, error) {
 
 // PreResponsePrompt returns the prompt before the response tag
 func (m *Model) PreResponsePrompt(p PromptVars) (string, error) {
```

Comment:
Can we just get rid of `PreResponsePrompt` and `PostResponsePrompt`? They just cause a lot of indirection.